spatialos-sdk/src/worker/entity_builder.rs

```diff
@@ -57,7 +57,7 @@ impl EntityBuilder {
         Ok(())
     }
 
-    pub fn add_handle<C: Component, T: Into<String>>(
+    pub fn add_handle<C: Component + Send, T: Into<String>>(
```

Comment:
What's the use case for exposing this in addition to `add` on the builder?

Reply:
This is part of the larger story for allowing users more control over when serialization happens (i.e. whether to serialize immediately or to use a vtable). The idea is that the `add` function will "do the right thing" based on your global configuration (i.e. use vtables if vtables are enabled, serialize immediately if not), and then you have `add_handle` and `add_serialized` to manually override that global configuration. This doesn't fully address the needs of users who want to work without vtable serialization, but I think it's a necessary part of supporting those use cases.